webui/react/src/App.tsx

```diff
@@ -54,6 +55,7 @@ const AppView: React.FC = () => {
   return (
     <div className={css.base}>
       {isAuthenticated && <NavBar username={username} />}
+      <AppContexts />
```

Comment:
nit: Move `<AppContexts />` above `<NavBar />`. This doesn't really affect functionality, but from a hierarchical standpoint, `isAuthenticated` and `username` are hydrated due to `AppContext`.